### PR TITLE
x11-misc/enlightenment-extra: new package

### PR DESCRIPTION
**Pull request by juippis**
Slowly trying to make Enlightenment a full suite in Gentoo. This is a program to install beatiful themes & backgrounds for Enlightenment. 

I'm aware of 
> /usr/share/applications/extra.desktop: error: value "0.0.1" for key "Version" in group "Desktop Entry" is not a known version

and reported it upstream via IRC, but it's a "fault" in desktop-file-validate rather than in their .desktop file. I'm gonna open a ticket in their bugsystem asking if the 'Version=' can be dropped. 

Didn't get QA warnings about 
`gnome2_icon_cache_update`
`xdg_desktop_database_update`
`xdg_mimeinfo_database_update`
so I didn't add them.

### CODE REVIEW

File: x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	local myconf=(
+		$(use_enable nls)
+	)
+
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	einstalldocs
+	emake install DESTDIR="${D}" || die "Installing ${PN} failed."
```
Comment (Zlogene) [inline x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild]: `default`?

File: x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+src_configure() {
+	local myconf=(
+		$(use_enable nls)
+	)
+
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	einstalldocs
```
Comment (Zlogene) [inline x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild]: then not needed

File: x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild (hunk 1 above)
Comment (juippis) [reply x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild]: Switched, thanks for review! Committed and rebased.

File: x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		$(use_enable nls)
+	)
+
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	default
+
+	emake install DESTDIR="${D}" || die "Installing ${PN} failed."
```
Comment (Zlogene) [inline x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild]: no, both `emake DESTDIR="${D}" install` and `einstalldocs` are being covered by `default`

Comment (juippis) [reply x11-misc/enlightenment-extra/enlightenment-extra-0.0.1.ebuild]: Ohh, okay. Seemed to work indeed, pushed an update.

**Comment by a17r** [comment]
> and reported it upstream via IRC, but it's a "fault" in desktop-file-validate rather than in their .desktop file.

the version is simply not meant to be used for the package version, this should be dropped upstream (patch it out for now).

**Comment by juippis** [comment]
Updated and rebased. Added a link to track upstream bug status in the ebuild, 
https://phab.enlightenment.org/T7167

**Pull request closed without merging**